app/models/user.rb

```diff
@@ -1,4 +1,7 @@
 class User < ActiveRecord::Base
+  validates :username, presence: true, uniqueness: true, length: { in: 3..15 },
+  format: { with: /[a-z0-9]/}, format: { without: /[^a-z0-9]/ }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside } missing.